src/screens/Music/stacks/components/SinglesListView.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useCallback } from 'react';
+import { Text, ScrollView, RefreshControl, StyleSheet, View } from 'react-native';
+import { useGetImage } from 'utility/JellyfinApi';
+import styled, { css } from 'styled-components/native';
+import { useNavigation } from '@react-navigation/native';
+import { useTypedSelector } from 'store';
+import { THEME_COLOR } from 'CONSTANTS';
+import TouchableHandler from 'components/TouchableHandler';
+import useCurrentTrack from 'utility/useCurrentTrack';
+import TrackPlayer from 'react-native-track-player';
+import Play from 'assets/play.svg';
+import Shuffle from 'assets/shuffle.svg';
+import useDefaultStyles from 'components/Colors';
+import usePlayTracks from 'utility/usePlayTracks';
+import { EntityId } from '@reduxjs/toolkit';
+import { WrappableButtonRow, WrappableButton } from 'components/WrappableButtonRow';
+import { MusicNavigationProp } from 'screens/Music/types';
+import DownloadIcon from 'components/DownloadIcon';
+import CloudDownArrow from 'assets/cloud-down-arrow.svg';
+import { useDispatch } from 'react-redux';
+import { queueTrackForDownload } from 'store/downloads/actions';
+import { selectDownloadedTracks } from 'store/downloads/selectors';
+import SingleImage from 'components/SingleImage';
+
+const styles = StyleSheet.create({
+    name: {
+        fontSize: 36, 
+        fontWeight: 'bold'
+    },
+    artist: {
+        fontSize: 24,
+        opacity: 0.5,
+        marginBottom: 12
+    },
+    index: {
+        width: 20,
+        opacity: 0.5,
+        marginRight: 5
+    },
+    artistTrackText:{
+        opacity: 0.5,
+        display: 'flex'
+    }
+});
+
+const TrackContainer = styled.View<{isPlaying: boolean}>`
+    padding: 5px 4px;
+    border-bottom-width: 1px;
+    flex-direction: row;
```

Comment:
If you add `align-items: center;` here, all the individual parts will be nicely aligned.